Portal.xcodeproj/project.pbxproj

```diff
@@ -315,6 +317,14 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		46952C311F4DB07F0075D536 /* Utils */ = {
```

Comment:
I would remove `Utils` directory and put `MessageHandler` inside `Renderers` directory